pppd/sys-linux.c

```diff
@@ -3083,79 +3069,12 @@ int sifvjcomp (int u, int vjcomp, int cidcomp, int maxcid)
 	return 1;
 }
 
-/********************************************************************
```

Comment:
You could frankly leave this in place despite the reference counting. New code would just call the function once for UP, and once for DOWN. And if that is done from one location, why not call the setifstate(u,1) directly. It would reduce the amount of code changed.

Reply:
also realized that I could just expose setifstate from sys-*, since netif_set_up and netif_set_down in both sys-linux and sys-solaris is identical.  This would result in the static keyword being dropped anyway, so can just drop the unused u (unit number?) argument regardless in order to avoid confusion as to what the argument really is, because dummy values were being passed in most cases.